### ISSUE TITLE
Option to not automatically localize 3rd party SCSS

### ISSUE THREAD

**Issue by ChrizzDF** [salsify/ember-css-modules#67]
Hi there,

Thanks for this really cool and helpfull addon.

I started an Ember app from scratch to understand this addon's behavior better and get used to it.
So far, so good. Currently I'm at the point where I installed [`ember-power-select`](https://github.com/cibernox/ember-power-select). I also figured out how to import those 3rd party SCSS styles globally ([`ecm-bootstrap`](https://github.com/dfreeman/ecm-bootstrap)) without being automatically localized by `ember-css-modules` but now I have all the styles for this power-select component in my CSS output:

The global styles are imported via:
```
// third-party.scss
@import "ember-power-select";

// app.scss
@after-module "./third-party";
```
which works nice!

But in the CSS output are also styles like:
```
._ember-power-select_4iaeql {
  position: relative; }

._ember-power-select_4iaeql *, ._ember-power-select-dropdown_4iaeql * {
  box-sizing: border-box; }

._ember-power-select-trigger_4iaeql {
  position: relative;
  border-top: 1px solid #aaaaaa;
  border-bottom: 1px solid #aaaaaa;
  border-right: 1px solid #aaaaaa;
  border-left: 1px solid #aaaaaa;
  border-radius: 4px;
  background-color: #ffffff;
  line-height: 1.75;
  overflow-x: hidden;
  text-overflow: ellipsis;
  min-height: 1.75em;
  user-select: none;
  -webkit-user-select: none;
  color: inherit; }
```

I think it's because `ember-css-modules` recognizes a new component (`power-select`through the `ember-power-select` addon) and therefore generates styles.

Is there a way to prevent automatically generating localized styles for addon components?
I'm sorry if there's a possibility that I've read over in the docs...

Thanks in advance! =)

**Comment by dfreeman** [comment]
Hi @ChrizzDF — in theory nothing from Ember Power Select should be getting localized. Out of the box Ember CLI provides a pretty nice framework for tools like `ember-css-modules` to only apply themselves in places they're wanted and not leak out into other dependencies.

Are you using any other custom PostCSS plugins like `postcss-import`? If not, then `ember-css-modules` shouldn't even be seeing the power-select styling (since Sass's `@import` would happen _after_ everything related to the modules).

Any chance you could link to a repo that reproduces the issue? That would be super helpful in tracking down whether there's just something odd in your configuration or an actual underlying bug on this end 🙂

**Comment by dfreeman** [comment]
Actually, another thought just occurred to me based on how I know ember-power-select makes its SCSS available in the `app` tree of the consumer. If I'm right, then I think this may actually already be fixed on a branch that's likely to become the next release of this addon. Could you try installing from the [lazy-engine-support branch](https://github.com/salsify/ember-css-modules/tree/lazy-engine-support) and seeing if you still run into the same issue?

**Comment by ChrizzDF** [comment]
Hey @dfreeman, thanks for your quick response!
Your assumption was confirmed in an [Issue](https://github.com/cibernox/ember-power-select/issues/870) I created - Ember Power Select does not import any styles automatically by default. =)

Hopefully I can test your latests suggestion with `lazy-engine-support` during this week :)

Thank again and I'll be back if there are new insights!

**Comment by dfreeman** [comment]
@ChrizzDF Just to let you know, the stuff on the `lazy-engine-support` branch is now on `master` and has been released as `0.7.0-beta.1`, if you want to give that a try.

**Comment by dfreeman** [comment]
@ChrizzDF I'm going to go ahead and close this out, since I believe it was fixed in 0.7.0 and never heard otherwise 🙂
